common/utils/src/main/resources/error/error-classes.json

```diff
@@ -589,6 +589,11 @@
           "<errors>"
```

Comment:
This is great! I was thinking upgrading the xml reader with data source v2 before but really stopped by the refactoring work involved. Thanks for adding it into the spark mainline to unify the interfaces and catch up with the main changes.

Reply:
Thanks for reviewing this PR.

Reply:
You're welcome! This is great!